snippets/java/fizzbuzz.md

```diff
@@ -0,0 +1,48 @@
+# Fizzbuzz
+
+## Code
+
+```java
+class FizzBuzz
+{
+    public static void main(String args[])
+    { 
+        int n = 100;
```

Comment:
can we have a better variable name here?

Reply:
I changed it to upper, is that fine?